plugins/woocommerce/package.json

```diff
@@ -225,6 +226,26 @@
 						"push"
 					]
 				},
+				{
+					"name": "Checks from plugin-check plugin",
+					"command": "test:plugincheck",
+					"optional": true,
+					"changes": [
+						"client/admin/config/*.json",
```

Comment:
Daily and release checks are not affected by file changes, so this can be empty. It won't hurt anything if you leave it like this, juts pointing out how it works.

Reply:
I had removed this, but forgot to push the change 🙁